Header header

std_msgs/Float32MultiArray bbox_stem
std_msgs/Float32MultiArray bbox_tomato
std_msgs/Float32MultiArray bbox_pedicel
std_msgs/Float32MultiArray bbox_sepal

std_msgs/Float32MultiArray mask_stem
std_msgs/Float32MultiArray mask_tomato
std_msgs/Float32MultiArray mask_pedicel
std_msgs/Float32MultiArray mask_sepal

# std_msgs/UInt8MultiArray mask_stem
# std_msgs/UInt8MultiArray mask_tomato
# std_msgs/UInt8MultiArray mask_pedicel
# std_msgs/UInt8MultiArray mask_sepal

